lib/ansiblelint/rules/PackageHasRetryRule.py

```diff
@@ -67,6 +67,16 @@ class PackageHasRetryRule(AnsibleLintRule):
         "zypper",
     ]
 
+    _module_ignore_states = [
+        "absent",
+    ]
+
     def matchtask(self, file, task):
-        return (task['action']['__ansible_module__'] in self.package_modules
-                and 'until' not in task)
+        module = task["action"]["__ansible_module__"]
+        if module in self._package_modules:
```

Comment:
```suggestion
        if module not in self._package_modules or 'until' in task:
            return False
```
will help keep 2 indentation levels less and will hugely improve readability.